src/main/scala/io/qbeast/table/IndexedTable.scala

```diff
@@ -515,14 +515,17 @@ private[table] class IndexedTableImpl(
               DeleteFile(
                 path = indexFile.path,
                 size = indexFile.size,
+                dataChange = false,
```

Comment:
DeleteFiles always have the dataChange to false?

Reply:
The default dataChange is always `true`. in this case, as it is in the `optimize()`, it must be `false`